src/components/Button/src/index.tsx

```diff
@@ -10,7 +10,7 @@ export interface ButtonProps extends BaseProps {
   /**
    * Simple click handler
    */
-  onClick?: () => void;
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
Does this work for touchscreens though? React.TouchEvent is also a typing, which I think will be passed to the onClick when touched instead of clicked, resulting in a typing error.

Reply:
Hmmm good point. I hadn't considered that. Will have a look into that